gluefactory/eval/utils.py

```diff
@@ -224,6 +262,36 @@ def eval_poses(pose_results, auc_ths, key, unit="°"):
     return summaries, best_th
 
 
+def eval_fundamental_matrices(fm_results, auc_ths, key, unit="°"):
+    pose_aucs = {}
+    best_th = -1
+    for th, results_i in fm_results.items():
+        pair_mean = []
+        for pair_results in results_i[key]:
+            pair_mean.append(AUCMetric(auc_ths, pair_results).compute())
+        pose_aucs[th] = np.array(pair_mean).mean(axis=0)
```

Comment:
It would be better to compute the AUC over epipolar errors in `eval_matches_epipolar_via_gt_points`, and then just average the respective value here. The only thing this function needs to compute is the mAA, where you can just average all entries in summary that start with `epi_error@X`.